### PR TITLE
Update --force flag help text

### PR DESCRIPTION
**Pull request by Marton6**

### CODE REVIEW

File: internals/secrethub/flags.go [hunk 1]
```diff
@@ -15,5 +15,5 @@ func registerTimestampFlag(r FlagRegisterer) *kingpin.FlagClause {
 }
 
 func registerForceFlag(r FlagRegisterer) *kingpin.FlagClause {
-	return r.Flag("force", "Ignore confirmation and fail instead of prompt for missing arguments.").Short('f')
+	return r.Flag("force", "Disable all confirmation prompts.").Short('f')
```
Comment (SimonBarendse) [inline internals/secrethub/flags.go]: Please include that this overwrites the output file if it already exists. See this ticket: https://github.com/secrethub/secrethub-cli/issues/261

Comment (Marton6) [reply internals/secrethub/flags.go]: Isn't that specific to the `secrethub inject` command?

Comment (SimonBarendse) [reply internals/secrethub/flags.go]: Yes correct. Should have been more explicit about that. My suggestion is to not use this function, but instead register the same flag with a more explicit message that explains exactly what it means for the `inject` command.

Comment (Marton6) [reply internals/secrethub/flags.go]: Got it, thanks!

Comment (SimonBarendse) [reply internals/secrethub/flags.go]: Please also include Marc's suggestion of making explicit that the `--force` flag is ignored if no `--output-file` is supplied. See the ticket: https://github.com/secrethub/secrethub-cli/issues/261#issuecomment-601125693

Comment (Marton6) [reply internals/secrethub/flags.go]: Done

**Pull request merged**